grammars/css.cson

```diff
@@ -483,7 +483,8 @@
         'include': '#comment-block'
       }
       {
-        'match': '--([A-Za-z0-9_-]+)'
+        'match': '--([A-Za-z][A-Za-z0-9_-]*)'
```

Comment:
Actually, even

```
--[\\w-]+
```

is too strict regarding the syntax, but since no one is really complaining about they want to use emoji or escaping in, like, class names, I think we're good.